packages/fxa-auth-server/test/scripts/stripe-products-and-plans-converter.js

```diff
@@ -256,38 +272,8 @@ describe('StripeProductsAndPlansConverter', () => {
     });
   });
 
-  describe('findLocaleStringFromStripePlan', () => {
-    it('returns a locale from the plan nickname if valid', () => {
-      const planWithLocalizedData = {
-        ...deepCopy(plan),
-        nickname: '123Done Pro Monthly es-ES',
-        metadata: {
-          'product:details:1': 'Producto nuevo',
-        },
-      };
-      const expected = 'es-ES';
-      const actual = converter.findLocaleStringFromStripePlan(
-        planWithLocalizedData
-      );
-      assert.equal(expected, actual);
-    });
-    it('returns null if no valid locale is found', () => {
-      const planWithLocalizedData = {
-        ...deepCopy(plan),
-        nickname: '123Done Pro Monthly',
-        metadata: {
-          'product:details:1': 'Producto nuevo',
-        },
-      };
-      const actual = converter.findLocaleStringFromStripePlan(
-        planWithLocalizedData
-      );
-      assert.isNull(actual);
-    });
-  });
-
   describe('stripePlanLocalesToProductConfigLocales', () => {
```

Comment:
Should we add some test cases where the product config and plan config are in the same language, and therefore the `locales` goes on the `planConfig` instead of the `productConfig`?

Reply:
Taking another look, and while I do see a new test case for `'re-throws an error from Google Translation API'`, I don't see one for putting the localized, English strings on the `planConfig`?